skimage/filters/ridges.py

```diff
@@ -413,18 +418,15 @@ def frangi(image, sigmas=range(1, 10, 2), scale_range=None, scale_step=None,
     # Generate empty (n+1)D arrays for storing auxiliary images filtered
     # at different (sigma) scales
     filtered_array = np.zeros(sigmas.shape + image.shape)
-    lambdas_array = np.zeros(sigmas.shape + image.shape)
+    lambdas_array = np.zeros_like(filtered_array)
 
     # Filtering for all (sigma) scales
     for i, sigma in enumerate(sigmas):
 
         # Calculate (abs sorted) eigenvalues
         lambda1, *lambdas = compute_hessian_eigenvalues(image, sigma,
-                                                        sorting='abs')
-
-        # Compute sensitivity to deviation from a plate-like structure
-        # see equations (11) and (15) in reference [1]_
-        r_a = np.inf if ndim == 2 else _divide_nonzero(*lambdas) ** 2
+                                                        sorting='abs',
+                                                        mode='reflect')
```

Comment:
Is there a reason mode is hardcoded to be `'reflect'`? I'd vote to have `mode` and `cval` be kwargs in `frangi` and pass them both through to `compute_hessian_eigenvalues`.

Reply:
I agree, good catch ;-)